Smartway.UiComponent/Inputs/SearchBar.xaml

```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentView xmlns="http://xamarin.com/schemas/2014/forms" 
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Smartway.UiComponent.Inputs.SearchBar"
+             x:Name="This">
```

Comment:
On a prit pour habitude de le nommer `Self` plutôt que `This`. On peut changer, met faut tous les changer dans ce cas :)

Reply:
Fait !